nexus/src/app/disk.rs

```diff
@@ -148,24 +151,35 @@ impl super::Nexus {
 
         // Reject disks where the size isn't at least
         // MIN_DISK_SIZE_BYTES
-        if params.size.to_bytes() < params::MIN_DISK_SIZE_BYTES as u64 {
+        if params.size.to_bytes() < MIN_DISK_SIZE_BYTES as u64 {
```

Comment:
any reason not to make it a `u64` and remove the `as u64` everywhere?

Reply:
Nope, that's probably fine.

Reply:
sounds like a reason!